Calibration/HcalAlCaRecoProducers/test/AlCaIsoTrackProducer_cfg.py

```diff
@@ -0,0 +1,35 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("ALCAISOTRACK")
+
+process.load("Configuration.Geometry.GeometryIdeal_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
```

Comment:
@bsunanda there is a mismatch either you use CondDB_v2 also for the `GlobalTag` or you change the `autoCond`